HerPortal.Data/Migrations/HerDbContextModelSnapshot.cs

```diff
@@ -21,6 +21,59 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
             NpgsqlModelBuilderExtensions.UseIdentityByDefaultColumns(modelBuilder);
 
+            modelBuilder.Entity("HerPortal.BusinessLogic.Models.LocalAuthority", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("integer");
+
+                    NpgsqlPropertyBuilderExtensions.UseIdentityByDefaultColumn(b.Property<int>("Id"));
+
+                    b.Property<string>("CustodianCode")
+                        .HasColumnType("text");
+
+                    b.Property<string>("Name")
+                        .HasColumnType("text");
+
+                    b.HasKey("Id");
+
+                    b.ToTable("LocalAuthorities");
+                });
+
+            modelBuilder.Entity("HerPortal.BusinessLogic.Models.User", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("integer");
+
+                    NpgsqlPropertyBuilderExtensions.UseIdentityByDefaultColumn(b.Property<int>("Id"));
+
+                    b.Property<string>("EmailAddress")
+                        .HasColumnType("text");
+
+                    b.Property<bool>("HasLoggedIn")
+                        .HasColumnType("boolean");
+
+                    b.HasKey("Id");
+
+                    b.ToTable("Users");
+                });
+
+            modelBuilder.Entity("LocalAuthorityUser", b =>

```

Comment:
I don't think you need this if you specify something like this on either the local authority or user tables:

```
 modelBuilder.Entity<LocalAuthority>()
        .HasMany(e => e.Users)
        .WithMany(e => e.LocalAuthorities);
```

https://learn.microsoft.com/en-us/ef/core/modeling/relationships/many-to-many

Reply:
The `DbContextModelSnapshot` is auto-generated, I use [Entity Framework conventions](https://www.entityframeworktutorial.net/code-first/configure-many-to-many-relationship-in-code-first.aspx) (also see [here](`BusinessLogic.Models.User`)) in `BusinessLogic.Models.User` and `BusinessLogic.Models.LocalAuthority` to replicate what you've done there with Fluent assertions 😊 

Even specifying that, EF still needs to generate the join table, so it shows up here!

Reply:
Sorry, my mistake, I thought I was looking at a different file!